functional/fixtures/units/single@.service

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStart=/bin/bash -c "while true; do echo Hello, World!; sleep 1; done"
```

Comment:
It could worth using [specifiers](https://github.com/coreos/fleet/blob/master/Documentation/unit-files-and-scheduling.md#systemd-specifiers), i.e.

```ini
ExecStart=/bin/bash -c "while true; do echo Hello, World %i!; sleep 1; done"
```